leaf-ui/rappid-extensions/AssignmentsTable.js

```diff
@@ -149,8 +149,7 @@ var AssignmentsTable = Backbone.View.extend({
      */
     displayAbsoluteIntentionAssignments: function() {
         this.model.getIntentions().forEach(intentionCell => {
```

Comment:
Change to intentionBbm just for clarity
```suggestion
        this.model.getIntentions().forEach(intentionBbm => {
```